### PR TITLE
Add gradle build documents

### LINKED ISSUE #58: Gradle build is not documented
We have gradle build but it's not documented. Please add it in how to build section.

### PR DESCRIPTION
**Pull request by birdstorm**
closes #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,79 +81,94 @@ Demo is avaliable in [KVRawClientTest](https://github.com/birdstorm/KVRawClientT
...
+After building, add following lines into your `pom.xml` if you are using Maven
+
 ```xml
 <dependency>
-    <groupId>org.tikv</groupId>
-    <artifactId>tikv-client-java</artifactId>
-    <version>2.0-SNAPSHOT</version>
+	<groupId>org.tikv</groupId>
+	<artifactId>tikv-client-java</artifactId>
+	<version>2.0-SNAPSHOT</version>
```
Comment (ilovesoup) [inline README.md]: Is that right? If it's not for a specific version, please make it clear.

File: README.md [hunk 2]
```diff
@@ -9,28 +9,59 @@ It is supposed to:
...
+gradle clean build -x test
+```
+
+To make a jar with dependencies
+
+```
+gradle clean fatJar -x test
+```
+
+The jar can be found in `./build/lib/`
```
Comment (ice1000) [inline README.md]: I suppose it's `libs`?

File: build.gradle [hunk 3]
```diff
@@ -15,47 +15,47 @@ tasks.withType(JavaCompile) {
...
+	main {
+		proto {
+			srcDir {'proto'}
+			include {'**/*.proto'}
+		}
+		java {
+			// include self written and generated code
+			srcDirs 'src/main/java', 'target/generated-sources/main/java' , 'target/generated-sources/main/grpc-java'
+		}
+	}
```
Comment (ice1000) [inline build.gradle]: I have to say that indentation changes are really annoying for code review

File: README.md (hunk 1 above)
Comment (birdstorm) [reply README.md]: it is used as local dependency, which does not have a specific version yet.

File: build.gradle (hunk 3 above)
Comment (birdstorm) [reply build.gradle]: yeah. The gradle file not correctly being formatted is also annoying.

**Comment by birdstorm** [comment]
@ice1000 PTAL

**Pull request merged**